src/glb-director-xdp/main.go

```diff
@@ -331,15 +340,19 @@ func (app *Application) ReloadForwardingTable() {
 		for b := uint32(0); b < uint32(srcTable.num_binds); b++ {
 			bindEntry := srcTable.binds[b]
 
-			bindKey := C.glb_bind{ipv4: C._get_bind_ipv4(&bindEntry), ipv6: C._get_bind_ipv6(&bindEntry), proto: C.ushort(bindEntry.proto), port: C._get_bind_port(&bindEntry)}
-			fmt.Printf("      bind: %v\n", bindKey)
-			tableIndex := uint32(i)
-			if err := tableMap4.Put(unsafe.Pointer(&bindKey), unsafe.Pointer(&tableIndex)); err != nil {
-				log.Fatal(err)
-			}
+            port_start := uint16(C._get_bind_port_start(&bindEntry))
```

Comment:
I think by eye the indentation might have gotten mixed up here, might be worth running it through `go fmt` to ensure it is standard.